backend/routes/reservation.js

```diff
@@ -208,4 +208,68 @@ router.delete('/single', async (req, res) => {
   res.json({ result: 'Deleted reservation' });
 });
 
+/**
+ * @swagger
+ *
+ * /reservation:
+ *   get:
+ *     description: Get a list of tables that are free for booking
+ *     produces:
+ *       - application/json
+ *     parameters:
+ *       - name: restaurantID
+ *         description: Primary Key of Restaurant database table
+ *         in: formData
+ *         required: true
+ *         type: integer
+ *       - name: numberOfGuests
+ *         description: The number of guests that the table is being booked for
+ *         in: formData
+ *         required: true
+ *         type: integer
+ *       - name: time
+ *         description: The start time of the booking
+ *         in: formData
+ *         required: true
+ *         type: string
+ *       - name: date
+ *         description: Date for when the booking is made
+ *         in: formData
+ *         required: true
+ *         type: date
```

Comment:
I believe this needs to be ```type: string``` to be able to test in the browser.

Reply:
Done